flaskbb/forum/views.py

```diff
@@ -1108,6 +1110,89 @@ def post(self, mode=None):
         preview = renderer(text)
         return preview
 
+def user_get_file_max_length():
+    user_groups_file_max_length=[current_user.primary_group.uploaded_file_max_length]
```

Comment:
```suggestion
    user_groups_file_max_length = current_user.primary_group.uploaded_file_max_length
    if user_groups_file_max_length == -1:
        return -1
```
Зачем так сложно считать, память не резиновая.
И еще этот -1 мех

Reply:
С -1 согласен, но брух у питухона не дефайнов